pkg/sql/alter_type.go

```diff
@@ -74,6 +75,27 @@ func (n *alterTypeNode) startExec(params runParams) error {
 func (p *planner) addEnumValue(
 	ctx context.Context, n *alterTypeNode, node *tree.AlterTypeAddValue,
 ) error {
+	if n.desc.Kind != descpb.TypeDescriptor_ENUM {
+		return pgerror.Newf(pgcode.WrongObjectType, "%q is not an enum", n.desc.Name)
+	}
+	// See if the value already exists in the enum or not.
+	found := false
+	for _, member := range n.desc.EnumMembers {
+		if member.LogicalRepresentation == node.NewVal {
+			found = true
```

Comment:
Not sure why it wasn't written this way in the first place, but we don't need a new variable and a break -- just do the notice/error return in here.

Reply:
my bad, i have updated this now.